cmake/emu/qemu.cmake

```diff
@@ -73,6 +73,9 @@ endif()
 # Connect monitor to the console chardev.
 list(APPEND QEMU_FLAGS -mon chardev=con,mode=readline)
 
+# Connect to other serial port
+list(APPEND QEMU_FLAGS ${QEMU_FLAGS_HOST_SERIAL})
```

Comment:
Why is a `QEMU_FLAGS_HOST_SERIAL` needed when we already have `QEMU_FLAGS_EXTRA` available ?